pull/github_test.go

```diff
@@ -777,3 +787,27 @@ func (c *MockGlobalCache) GetPushedAt(repoID int64, sha string) (time.Time, bool
 func (c *MockGlobalCache) SetPushedAt(repoID int64, sha string, t time.Time) {
 	c.PushedAt[fmt.Sprintf("%d:%s", repoID, sha)] = t
 }
+
+// PathAndQueryMatcher matches both path and query parameters
+type PathAndQueryMatcher struct {
```

Comment:
Let's put this in `pull/responseplayer_test.go` with the other matcher implementations, since it is pretty generic